### DISCUSSION TITLE
Allow passing additional fetch parameters, like signal for aborting requests

### DISCUSSION THREAD

**Discussion by iSplasher** [supabase/supabase#17715] [Feature Requests]
Please make it possible to pass additional parameters (mostly an AbortSignal) to the fetch request, like this `superbase.functions.invoke("somefunction", { signal })`

**Comment by kasrak** [comment]
Just ran into this as well, it'd be great to be able to abort edge function invocations.

Maintainers: would you be open to accepting a PR for this change?

**Reply by laktek** [reply]
yep, please submit a PR to functions-js

**Reply by dsc8x** [reply]
Here we go https://github.com/supabase/functions-js/pull/93

**Reply by jens-f** [reply]
Would it be possible to get the PR above reviewed and merged? We would also require this functionality.

**Reply by anngbaum** [reply]
just ran into this and would add a +1 here – would be a big help
